alfresco-transform-core-aio/alfresco-transform-core-aio/src/main/java/org/alfresco/transformer/transformers/LibreOfficeAdapter.java

```diff
@@ -1,56 +1,57 @@
-/*

```

Comment:
Fix your git/ide config so that we only see the actual diff (check with @PunchedChimera).

Note: EOL/formatting/whitespace changes should be reserved for separate PRs.